src/utils.h

```diff
@@ -96,6 +98,8 @@ extern TSDLLEXPORT List *ts_get_reloptions(Oid relid);
 
 #define STRUCT_FROM_TUPLE(tuple, mctx, to_type, form_type)                                         \
 	(to_type *) ts_create_struct_from_tuple(tuple, mctx, sizeof(to_type), sizeof(form_type));
```

Comment:
I don't think it used anywhere as it is replaced with `STRUCT_FROM_SLOT`. Can it be removed?

Reply:
I'll take a look